go-controller/pkg/ovn/secondary_layer3_network_controller_test.go

```diff
@@ -369,6 +385,10 @@ var _ = Describe("OVN Multi-Homed pod operations", func() {
 			dummyPrimaryLayer3UserDefinedNetwork("192.168.0.0/16", "192.168.1.0/24"),
 			icClusterWithDisableSNATTestConfiguration(),
 		),
+		Entry("pod on a user defined primary network on an IC cluster with EgressFirewall enabled",
+			dummyPrimaryLayer3UserDefinedNetwork("192.168.0.0/16", "192.168.1.0/24"),
+			icClusterEnableEgressFirewallTestConfiguration(),
```

Comment:
Isn't this better to be {EFW:true, IC: true, whatever else is in minimal config to be true?} here and in other It functions so that readers can understand it in one go what is being performed here? the abstraction seems overengineerd for enabling the booleans but I am happy to stand corrected if its only me who feels this eay @pperiyasamy what do you think?

Reply:
yes @tssurya , i agree with your concern, this would bloat config util with too many methods for different configs. doing it now opts argument on the base method. hope you're fine with it now.